tdrs-backend/tdpservice/users/api/login.py

```diff
@@ -62,19 +62,51 @@ def handle_user(self, request, id_token, decoded_payload):
         if "token" not in request.session:
             request.session["token"] = id_token
 
+        # Authenticate users with the unique "subject" `sub` UUID from the payload.
+        sub = decoded_payload["sub"]
+        email = decoded_payload["email"]
+
+        # First account for the initial superuser
+        if (su_username := os.environ.get('DJANGO_SU_NAME')) and su_username == email:
```

Comment:
There is an edge case where we need to save the unique `sub` value of the initial superuser defined by `DJANGO_SU_NAME`. In this branch, we must therefore authenticate with the email. All other authentications will later be done using this `sub` value: `user_id=sub`.